tools/buildutils/build_packages.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-
-set -e -x
```

Comment:
Had to move this line down a bit, because one of the two `which` statements will always produce an error. I could `set -x` in line 3 already, but `set -e` only works below thes `which` statements. Else I'd need some error-free alternate way to tell apart the package manager - or split it into two files, so that this check becomes irrelevant.
https://github.com/google/android-cuttlefish/pull/613/commits/8871e570756a4ccb4352dfcce7a1ca1970c60f48

Reply:
Replaced that check altogether, by checking for presence of file `/bin/dnf` - and defaulting to Debian. https://github.com/google/android-cuttlefish/pull/613/commits/4238336a7fc097e720f26aad65eac566664d4fb6